src/Config/HotReloadEventHandler.cs

```diff
@@ -26,10 +26,6 @@ public HotReloadEventHandler()
             { METADATA_PROVIDER_FACTORY_ON_CONFIG_CHANGED, null },
             { QUERY_ENGINE_FACTORY_ON_CONFIG_CHANGED,null },
             { MUTATION_ENGINE_FACTORY_ON_CONFIG_CHANGED,null },
-            { QUERY_EXECUTOR_ON_CONFIG_CHANGED, null },
-            { MSSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, null },
-            { MYSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, null },
-            { POSTGRESQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, null },
```

Comment:
Can you add comment surrounding `QUERY_MANAGER_FACTORY_ON_CONFIG_CHANGED` here that describes how the "query executors since those are already being correctly refreshed already by the query manager factory reload process."